Adds tinted versions of the "colorIndex" neutral colors.

phuson opened the pull request:
![screen shot 2016-04-19 at 4 58 27 pm](https://cloud.githubusercontent.com/assets/7669/14659360/371b8206-0650-11e6-9bfc-d502a0a8b724.png)

I decided to go with appending the `-tint-1` for the colorIndex names so that in the future, we can add more tinted versions.

Let me know if I'm heading in the right direction with this, I can go ahead and add the tinted versions to the rest of the colors (accent, light, etc.).

ericsoderberghp commented:
I get the spirit of the "-tint-1" name but I'm thinking we don't need to design for the future quite so much. I think just the "t" suffix will align better with the existing "a" suffix. If we end up needing to extend to later for additional alpha values or tints, we'll handle it then. Otherwise, these changes look fine.

phuson commented:
Okay, updated to just use the `-t` suffix. Also added the tinted versions for the `accent` set of colors. I'm not sure if we need it for `light`, `grey`, and `graph`. Please let me know if this is good to go.

![screen shot 2016-04-20 at 9 51 16 am](https://cloud.githubusercontent.com/assets/7669/14683113/e7654c40-06dd-11e6-83c9-eb51b162eefa.png)
